app/theme/index.ts

```diff
@@ -1,6 +1,22 @@
 import { extendTheme, withDefaultColorScheme } from '@chakra-ui/react';
 import colors from './colors';
 
-const theme = extendTheme({ colors }, withDefaultColorScheme({ colorScheme: 'brand' }));
+const theme = extendTheme(
+  {
+    colors,
+    styles: {
+      global: {
+        '.domain-form': {
+          // InfoIcon
+          '.chakra-icon': {
+            color: '#d9d9d9',
```

Comment:
I would use the colour value defined value in `./colors` here.

Reply:
I decided to not add another value to that file because from what I understand, I would need to define a new property just for one color value

Reply:
I just pointed that out because, let's say, we have to change colours to a lighter/darker shade later on; we could update values defined in the `./colors` file, which should be enough. In such cases, we could easily miss updating the value of this icon if we are not careful enough. However, if we are confident we would never be required to change the colour scheme to a darker/lighter shade, then hardcoding here is fine.

Reply:
So, the issue with using one of the existing colors is that, none of the colors in the `./colors` file match this color. Should I add another object for gray colors?
```ts
const colors = {
  brand: {
    50: '#ffe4e3',
    100: '#feb7b6',
    ...
  },
  gray: {
    ...
  }
};
```

Reply:
@Myrfion @Ririio What would you guys suggest?

Reply:
I would add the default values and add a new instance for the gray color. Maybe like gray.350, should follow the format though from light to darkest

Reply:
Should I add an entire color palate in this case based around the light grayish color of the icon?

Reply:
I think that's how it works

Reply:
I could not get it to work with `gray` as it overrides existing colors for a lot of components. So, I added a different name: `brand_gray`. If this looks weird, we can revert to manually overriding the color value for this one icon only